File: 944-smallest-range-i/smallest-range-i.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var smallestRangeI = function(nums, k) {
    min = Math.min(...nums);
    max = Math.max(...nums);
    return Math.max(0,(max-k)-(min+k));
};

File: 944-smallest-range-i/smallest-range-i.java
class Solution {
    public int smallestRangeI(int[] nums, int k) {
       
        int min=Arrays.stream(nums).min().getAsInt();
        int max=Arrays.stream(nums).max().getAsInt();
    
        return (max-k)-(min+k)<0?0:(max-k)-(min+k);
    }
}